bin/ch-run.c

```diff
@@ -95,7 +96,7 @@ void privs_verify_invoking();
 
 const struct argp argp = { options, parse_opt, args_doc, usage };
 extern char **environ;  // see environ(7)
-
+char *sqfs_unpack = "/var/tmp";
```

Comment:
I wonder if the default mount point should be `/var/tmp/$USER/ch-mnt`? Or maybe we should change the default image storage to something like `/var/tmp/$USER.ch/ch-image` and then for this use `/var/tmp/$USER.ch/mnt`? The difference is that `/var/tmp/$USER` isn't Charliecloud specific while `/var/tmp/$USER.ch` is.